tsconfig.json

```diff
@@ -1,15 +1,15 @@
 {
-  "extends": "@tsconfig/node14/tsconfig.json",
   "compilerOptions": {
-    "lib": ["es2020"],
+    "lib": ["esnext"],
```

Comment:
Did these end up being required for deployment to work?

Reply:
Nope.....the solution was crazy.

Reply:
Reverted